.travis.yml

```diff
@@ -3,10 +3,11 @@ language: node_js
 node_js:
   - "0.10"
   - "4"
+  - "5"
 
 matrix:
   allow_failures:
-    - node_js: "4"
```

Comment:
@anhou and @benbp all of them currently pass on Node 4 with most also passing on Node 5.